src/main/java/models/Computer.java

```diff
@@ -0,0 +1,17 @@
+package models;
+
+/**
+ * Created by isaac on 3/10/16.
+ */
+public class Computer extends Game
+{
+
+    //build deck or something? Kanna make it better
+    public Computer()
+    {
+        this.buildDeck();
```

Comment:
I think this is why your test is failing. You are calling buildDeck() here and in Game.java